package.json

```diff
@@ -64,7 +68,10 @@
     "ts-prune": "^0.10.3",
     "tsx": "^3.9.0",
     "turbo": "^1.6.3",
-    "typescript": "^4.8.3"
+    "typescript": "^4.8.3",
+    "vitest": "^0.23.4",
```

Comment:
Shouldn't all these deps be in packages/test?

Reply:
I'm adding back Vitest to the root `package.json`, because the types for the globals (`vi.mock`, `vi.fn`, ...) need to be declared inside `tsconfig.json` (either at the root, or inside each package - but that would mean duplication)